Workbooks/SapMonitor2.0/PrometheusHaCluster/PrometheusHaCluster.workbook

```diff
@@ -0,0 +1,1215 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "3b07e0eb-f988-40ef-bd30-b56f8bac5d2e",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_alert_templates",
+            "type": 1,
+            "isRequired": true,
+            "value": "[\n\t{\n\t\t\"name\": \"Cluster Primary node switched\",\n\t\t\"description\": \"SUSE Linux only: Fired when primary node switched for a resource\",\n\t\t\"author\": \"Microsoft\",\n\t\t\"templateId\": \"hacluster-resource-switched\",\n\t\t\"severity\": \"2\",\n\t\t\"alertTemplate\": {\n\t\t\t\"query\": \"Prometheus_HaClusterExporter_CL| where name_s == 'ha_cluster_pacemaker_resources'| where value_d == 1| extend jsonlabel = parse_json(labels_s)| extend PrimaryNode = tostring(jsonlabel.node), resource = tostring(jsonlabel.resource), clone = tostring(jsonlabel.clone), ismaster = tostring(jsonlabel.role)| extend isprimary = iff(clone == '', 1, iff(ismaster == 'master', 1, 0))| where isprimary == 1| summarize count() by bin(TimeGenerated, 30m), resource, clusterName_s, sid_s, PrimaryNode| summarize hostcount=count() by TimeGenerated, resource, clusterName_s, sid_s| extend AggregatedValue = iff(hostcount > 1, 1 , 0)\",\n\t\t\t\"thresholdOperator\": \"GreaterThan\",\n\t\t\t\"defaultThreshold\": \"0\",\n\t\t\t\"metricMeasurement\": {\n\t\t\t\t\"thresholdOperator\": \"GreaterThan\",\n\t\t\t\t\"threshold\": \"0\",\n\t\t\t\t\"metricTriggerType\": \"Consecutive\",\n\t\t\t\t\"metricColumn\": \"AggregatedValue\",\n\t\t\t\t\"frequencyInMinutes\": \"10\",\n\t\t\t\t\"timeWindowInMinutes\": \"60\"\n\t\t\t}\n\t\t}\n\t}\n]\n",
+            "typeSettings": {
+              "multiLineText": true,
+              "editorLanguage": "json",
+              "multiLineHeight": 20
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          },
+          {
+            "id": "6fcf88fd-b02d-44d4-8518-dd98212aedae",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_provider_type",
+            "type": 1,
+            "value": "PrometheusHaCluster",
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          },
+          {
+            "id": "92e2c347-22df-4b92-9de7-54a4088c3929",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_provider_instance",
+            "type": 1,
+            "value": "HACLUSTER"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.hanaonazure/sapmonitors"
```

Comment:
This needs to be changed to the one we have in v2.

Reply:
This is corrected.